sound/soc/soc-core.c

```diff
@@ -785,10 +802,10 @@ int snd_soc_resume(struct device *dev)
 
 	/* activate pins from sleep state */
 	for_each_card_rtds(card, rtd) {
-		struct snd_soc_dai *cpu_dai = rtd->cpu_dai;
-
-		if (cpu_dai->active)
-			pinctrl_pm_select_default_state(cpu_dai->dev);
+		for_each_rtd_codec_dai(rtd, i, cpu_dai) {
```

Comment:
Looks like typo here as well? For each cpu dai?

Reply:
Indeed. Thanks.